features/steps/jobs_steps.go

```diff
@@ -0,0 +1,154 @@
+package steps
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	componenttest "github.com/ONSdigital/dp-component-test"
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	"github.com/ONSdigital/dp-search-reindex-api/config"
+	"github.com/ONSdigital/dp-search-reindex-api/models"
+	"github.com/ONSdigital/dp-search-reindex-api/service"
+	"github.com/ONSdigital/dp-search-reindex-api/service/mock"
+	"github.com/cucumber/godog"
+	"github.com/pkg/errors"
+	"github.com/rdumont/assistdog"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type JobsFeature struct {
+	ErrorFeature   componenttest.ErrorFeature
+	svc            *service.Service
+	errorChan      chan error
+	Config         *config.Config
+	HTTPServer     *http.Server
+	ServiceRunning bool
+	ApiFeature     *componenttest.APIFeature
+	responseBody   []byte
+}
+
+func NewJobsFeature() (*JobsFeature, error) {
+	f := &JobsFeature{
+		HTTPServer:     &http.Server{},
+		errorChan:      make(chan error),
```

Comment:
does this need a buffer size?

Reply:
I'm not sure. It works ok with the default of no buffer size but I'm happy to put a size in if there's one you would suggest.

Reply:
I'm not sure tbh - if its working as expected I am happy with how it is.